### PR TITLE
New version for openCARP packages, v14.0

### PR DESCRIPTION
**Pull request by MarieHouillon**
Add a new version for the packages of the openCARP ecosystem, opencarp, meshtool and py-carputils.

### CODE REVIEW

File: var/spack/repos/builtin/packages/meshtool/package.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ class Meshtool(MakefilePackage):
 
     version("master", branch="master", preferred=True)
     # Version to use with openCARP releases
+    version("oc14.0", commit="867431d")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/meshtool/package.py:19]: Can we use the full commit hash? Why do most versions have the same commit hash?

File: var/spack/repos/builtin/packages/opencarp/package.py [hunk 2]
```diff
@@ -46,6 +46,7 @@ class Opencarp(CMakePackage):
     version(
         "7.0", commit="78da91952738b45760bcbc34610814a83c8c6299", submodules=False, no_cache=True
     )
+    version("14.0", commit="a92e564a", submodules=False, no_cache=True, preferred=True)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/opencarp/package.py:49]: Can we sort these from newest to oldest and use the full hash?

File: var/spack/repos/builtin/packages/opencarp/package.py [hunk 3]
```diff
@@ -66,7 +67,7 @@ class Opencarp(CMakePackage):
     depends_on("py-carputils", when="+carputils", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/opencarp/package.py:67]: The above dep types seem odd. We're linking to the pkgconfig/python/perl libs? Should they be type=build? Or run?

**Review by adamjstewart — COMMENTED** [review]
Python package looks good

**Comment by alalazo** [comment]
Closing, since there has been no activity in a while. Feel free to reopen if you want to continue work on this.

**Pull request closed without merging**